apluslms-yamlidator/apluslms_yamlidator/utils/__init__.py

```diff
@@ -0,0 +1,12 @@
+BOOLEANS = {
+    0: False, 1: True,
+    '0': False, '1': True,
+    'false': False, 'true': True,
+    'False': False, 'True': True
+}
+
+def convert_to_boolean(val):
+    if val not in BOOLEANS:
+        raise ValueError
```

Comment:
A bit more generic version would be `raise ValueError("invalid literal for convert_to_boolean(): %r" % (val,))`. Would it be nice?